file-formats/cota/cota/README.md

```diff
@@ -0,0 +1,5 @@
+# COTA File Format
+
+File | Cardinality | Definition | Schema | Example
+:--- | :---  | :--- | :--- | :---
+`<name>.cota.json` | 1 | [`zif_aff_cota_v1.intf.abap`](./type/zif_aff_cota_v1.intf.abap) | [`cota-v1.json`](./cota-v1.json) | [`z_contarget_example.cota.json`](./examples/z_contarget_example.cota.json)
```

Comment:
As the example does not exist, the hyperlink is broken and results in
<img width="863" alt="Screenshot 2023-07-21 at 11 47 36" src="https://github.com/SAP/abap-file-formats/assets/38354196/4dcc367d-d80f-41c0-bc58-aa65e6df7332">

Reply:
Issue was generated https://github.com/SAP/abap-file-formats/issues/544

Reply:
I think, we should remove also the example from the readme as long as we don't have the example. Otherwise the checks will continue to fail ;)

```suggestion
`<name>.cota.json` | 1 | [`zif_aff_cota_v1.intf.abap`](./type/zif_aff_cota_v1.intf.abap) | [`cota-v1.json`](./cota-v1.json) | 
```